clientv3/util/util.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
```

Comment:
I thought about that, but let me make the case against it:

1) You should do:

```
	clientv3util "github.com/coreos/etcd/clientv3/util"
```

if you want it named that way.

2) If you had it named `clientv3util` this looks like:

```
	"github.com/coreos/etcd/clientv3/clientv3util"
```

aka department of redundancy department. and renames of clientv3 cause lots of carnage below.